lib/resources.js

```diff
@@ -28,6 +28,10 @@ exports.categories = {
         'Walmart Mobile node.js setup': {
             url: 'https://gist.github.com/hueniverse/7686452',
             description: 'An insight into how hapi.js is used at Walmart'
+        },
+        'rate-limiter-flexible-plugin-example': {
```

Comment:
Just put it in alphabetical order please, otherwise LGTM

Reply:
Thanks